src/dashboard/frontend/transfer-browser/app/services/transfer.service.js

```diff
@@ -17,6 +17,7 @@ class Transfer {
     this.type = 'standard';
     this.accession = '';
     this.components = [];
+    this.auto_approve = false;
```

Comment:
The transfer browser widget prefers the behaviour to be set to false by default because otherwise we'd be introducing a backward incompatible change.